com.github.treagod.spectator.json

```diff
@@ -0,0 +1,68 @@
+{
+  "app-id": "com.github.treagod.spectator",
+  "base": "io.elementary.BaseApp",
+  "base-version": "juno-20.08",
+  "runtime": "org.gnome.Platform",
+  "sdk": "org.gnome.Sdk",
+  "runtime-version": "3.38",
+  "command": "com.github.treagod.spectator",
+  "finish-args": [
+    "--filesystem=home",
```

Comment:
What does it need home access for?

Reply:
The application saves the user data in $HOME/.local/share/com.github.treagod.spectator/com.github.treagod.spectator.db

Reply:
You should use `$XDG_DATA_HOME` instead, flatpak grants access there automatically.

Reply:
Is it ok if I use `$XDG_DATA_HOME`  in my application if it is defined and `$HOME/.local/share/com.github.treagod.spectator/com.github.treagod.spectator.db` if it is not defined?

Reply:
That sounds good, Flatpak ensures it's always defined though but a fallback doesn't hurt.